app/lib/new_references_feature.rb

```diff
@@ -0,0 +1,16 @@
+class NewReferencesFeature
+  attr_reader :application_form
+  OLD_REFERENCE_FLOW_CYCLE_YEAR = 2022
+
+  def initialize(application_form)
+    @application_form = application_form
+  end
+
+  def active?
+    FeatureFlag.active?(:new_references_flow) && application_form.recruitment_cycle_year > OLD_REFERENCE_FLOW_CYCLE_YEAR
+  end
+
+  def inactive?
```

Comment:
This is a confusing method as it's not the inverse of `active?` as it doesn't evaluate the feature flag. Is it necessary? Why not `!active?` ?

Reply:
It is confusing but this is because of Apply 2:

So when we turn on the feature flag in time for the Apply 1 deadline, they will all carry over into the new flow (and be moved into the new recruitment cycle year).

Meanwhile we will have candidates who are in Apply 2 – so will still need to see the old flow – until they hit the A2 deadline two weeks later.

So we need both reference flows to be running in parallel for that two week period

Reply:
As discussed earlier we should rename this method, it's not the opposite of active?
Also I'm confused why this was comment/conversation was marked as resolved given the discussion about this PR earlier - ideally we should not even have this class as the `ApplicationForm` now has the same logic doesn't it?